src/partialBindTo.ts

```diff
@@ -0,0 +1,74 @@
+import { purry } from './purry';
+
+type RemovePrefix<
+  P extends ReadonlyArray<unknown>,
+  T extends ReadonlyArray<unknown>,
+> = P extends readonly [...T, ...infer U]
+  ? U
+  : ["ERROR(partialBindTo): Data doesn't match function signature."];
+
+/**
+ * Creates a function that calls `func` with `data` put before the arguments
+ * it receives.
+ *
+ * @param data the arguments to put before
+ * @param func the function to wrap
+ * @returns a partially bound function
+ *
+ * @signature
+ *    R.partialBindTo(data, func)
+ *
+ * @example
+ *    const fn = (x, y, z) => `${x}, ${y}, and ${z}`
+ *    const partialFn = R.partialBindTo([1, 2], fn)
+ *    partialFn(3) // => 1, 2, and 3
+ *
+ * @dataFirst
+ * @category Function
+ * @see partialRightBindTo
+ */
+export function partialBindTo<
+  T extends ReadonlyArray<unknown>,
+  F extends (...args: any) => any,
+>(
+  data: [...T],
```

Comment:
Use our `IterableContainer` utility type. I can't explain why it works differently, but it seems to have something to do with how typescript infers tuples. I got the inspiration from:

https://github.com/microsoft/TypeScript/blob/2a0edf7949ad8427e3ef233a4c549ecbe32953e1/src/lib/es2015.promise.d.ts#L21

```suggestion
  T extends IterableContainer,
  F extends (...args: any) => any,
>(
  data: T,
```